android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/transition/effect/Gravity.kt

```diff
@@ -0,0 +1,12 @@
+package com.badoo.ribs.core.routing.transition.effect
+
+enum class Gravity {
+    LEFT, RIGHT, TOP, BOTTOM;
```

Comment:
LEFT -> START
RIGHT -> END

In case if in future you will decide to support RTL)

Reply:
But that would be a lie now, since we don't actually support that. If we'll add support, START and END should be new elements in addition to the current ones, not their replacements.